spark/v3.1/spark/src/test/java/org/apache/iceberg/spark/source/TestIcebergSourceTablesBase.java

```diff
@@ -2028,4 +2042,10 @@ public static Dataset<Row> selectNonDerived(Dataset<Row> metadataTable) {
   public static Types.StructType nonDerivedSchema(Dataset<Row> metadataTable) {
     return SparkSchemaUtil.convert(selectNonDerived(metadataTable).schema()).asStruct();
   }
+
+  private long getDataFileSizeInBytes(Iterable<DataFile> dataFiles) {
```

Comment:
nit: we can remove 'get' (Iceberg code style guideline are a bit different:  https://iceberg.apache.org/contribute/#method-naming)